sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1081,6 +1081,17 @@ object RapidsConf {
       .checkValue(v => v > 0, "The maximum number of files must be greater than 0.")
       .createWithDefault(Integer.MAX_VALUE)
 
+  val ENABLE_DELTA = conf("spark.rapids.sql.format.delta.enabled")
+      .doc("When set to false disables all Delta Lake acceleration")
```

Comment:
nit: can we say read and write acceleration? This is minor. It is just that we still support zorder optimization.

Reply:
Since Delta Lake reads aren't really controlled by this top-level enable, decided to remove it entirely.  If and when we support something explicit for Delta Lake other than writes that we want to control with a top-level enable, we can re-introduce it.